NAMESPACE

```diff
@@ -1,6 +1,11 @@
 # Generated by roxygen2: do not edit by hand
 
+export(dxdt)
+export(powerlawA)
 export(simulateGLV)
+export(simulateSOI)
+export(update_propensities)
```

Comment:
In our coding style, this should be something like `updatePropensities`.

Let's be consistent.